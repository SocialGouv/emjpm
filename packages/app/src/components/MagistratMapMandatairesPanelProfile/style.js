const MagistratMandatairePanelStyle = {
  width: "600px",
  height: "100%"
};

export { MagistratMandatairePanelStyle };
