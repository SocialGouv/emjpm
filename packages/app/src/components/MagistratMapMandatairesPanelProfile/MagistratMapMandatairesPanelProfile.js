import { Button } from "@socialgouv/emjpm-ui-core";
import React, { useContext } from "react";
import { Box } from "rebass";

import { MapContext } from "../MagistratMapMandataires/context";
import { MagistratMandatairePanelStyle } from "./style";

const MagistratMapMandatairesPanelProfile = props => {
  // const { currentGestionnaire } = props;
  const { setcurrentGestionnaire } = useContext(MapContext);

  const close = () =>
    setcurrentGestionnaire({
      isActive: false,
      latitude: 48.8534,
      longitude: 2.3488,
      currentId: null,
      currentDiscriminator: null
    });
  return (
    <Box sx={MagistratMandatairePanelStyle} {...props}>
      <Button onClick={() => close()}>Fermer</Button>
      MagistratMandatairePanel
    </Box>
  );
};

export { MagistratMapMandatairesPanelProfile };
