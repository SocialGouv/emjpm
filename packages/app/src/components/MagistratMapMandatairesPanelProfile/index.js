import { MagistratMapMandatairesPanelProfile } from "./MagistratMapMandatairesPanelProfile";

export { MagistratMapMandatairesPanelProfile };
