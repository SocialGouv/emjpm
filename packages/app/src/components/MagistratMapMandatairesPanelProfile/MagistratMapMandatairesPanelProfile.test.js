import React from "react";
import renderer from "react-test-renderer";

import { MagistratMapMandatairesPanelProfile } from "./MagistratMapMandatairesPanelProfile";

it("renders correctly", () => {
  const tree = renderer.create(<MagistratMapMandatairesPanelProfile />).toJSON();
  expect(tree).toMatchSnapshot();
});
