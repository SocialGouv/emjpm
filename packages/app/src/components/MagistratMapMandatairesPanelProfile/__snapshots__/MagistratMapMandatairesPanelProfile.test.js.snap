// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-qocjfr"
>
  <button
    className="css-clrcon"
    onClick={[Function]}
  >
    <div
      className="css-o6z23g"
    >
      Fermer
    </div>
  </button>
  MagistratMandatairePanel
</div>
`;
